### PR TITLE
made Readme.md more descriptive

### PR DESCRIPTION
**Pull request by gautamkrishnar**
README.md is the face of any projects, you will have to make sure that it contain relevant information about your project including its dependencies, how to use instructions, team mates name etc.... 

Improved README.md
* Added some info from Wikipedia
* Left some notes

@Aksh77 @Poirot1729  Please check my notes (`<!-- note -->`) in the readme file's comments and try to make it more awesome 😄 ❤️  :octocat:

### CODE REVIEW

**Comment by gautamkrishnar** [comment]
👍

**Comment by gautamkrishnar** [comment]
@Aksh77  sorry i forgot to edit something....

**Comment by Aksh77** [comment]
@gautamkrishnar Thanks a lot for your contributions and the suggestions :smile:

File: README.md [hunk 1]
```diff
@@ -1 +1,35 @@
...
+
+<!-- todo @Aksh77 Add some info about your project after replacing this line -->
+
+### How to use
+* Just clone the repo, use:
+ ```bash
+ git clone https://github.com/Aksh77/ACM-Team-Notebook.git
+ ```
+ 
+* Browse to the ACM Notedbook directory:
```
Comment (gautamkrishnar) [inline README.md:13]: @Aksh77 please fix this typo

File: README.md [hunk 2]
```diff
@@ -1 +1,35 @@
...
+ ```bash
+ cd ACM-Team-Notebook
+ ```
+* Compile and run the programs:
```
Comment (gautamkrishnar) [inline README.md:17]: @Aksh77 just insert a newline here....

File: README.md (hunk 1 above)
Comment (Aksh77) [reply README.md:13]: Had noted that already. Will fix it. :+1: 
Thanks :smile:

Comment (gautamkrishnar) [reply README.md:13]: 😄

**Pull request merged**